### PR TITLE
Remove route markers and search capability

### PR DESCRIPTION
**Pull request by mhendri**
Branch adds the ability to remove waypoints/markers from the map. Also adds a geocode feature to search start and end points for route.

### CODE REVIEW

File: app/static/js/app.js [hunk 1]
```diff
@@ -20,17 +20,18 @@ $.getJSON('/static/geo_data/borough_zip_geo.json', function(data) {
...
     var btn = L.DomUtil.create('button', '', container);
     btn.setAttribute('type', 'button');
     btn.innerHTML = label;
     return btn;
   }
 
+// Begin: map and control for click to add waypoints
 var control = L.Routing.control({
-      waypoints: []
+      waypoints: [],
```
Comment (johnnyplaydrums) [inline app/static/js/app.js]: @mhendri should that comma be there if there are no attributes after it?

**Comment by mhendri** [comment]
Ever the intrepid investigator, John! That right there is an artifact and does not belong there. Good eye!

**Comment by johnnyplaydrums** [comment]
@mhendri DTM (down to merge) once that's fixed!

**Comment by johnnyplaydrums** [comment]
🚀

**Comment by mhendri** [comment]
Merging!

**Pull request merged**